cloudformation-prod.json

```diff
@@ -96,7 +94,7 @@
 		"Ec2Instance" : {
       "Type" : "AWS::EC2::Instance",
       "Properties" : {
-        "ImageId" : "ami-0d77397e",
+        "ImageId" : { "Fn::FindInMap" : [ "RegionMap", { "Ref" : "AWS::Region" }]},
         "KeyName" : { "Ref" : "KeyName" },
 				"InstanceType" : "t2.micro",
```

Comment:
Yes, when we are building our final deployment we should set this to something reasonable.  Also this can be a parameter to cloudformation.